modules/microshift-data-cleaning-overview.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-overview_{context}"]
+= Data cleanup script overview
+
+See the usage and available options of the `microshift-cleanup-data` script by running the script without arguments.
+
+.Procedure
+
+. Run the following script.
```

Comment:
```suggestion
. Run the following script.
```
List the usage and available options of the `microshift-cleanup-data` script by running the following command: